spec/integration/data_stream/manifest.spec.yml

```diff
@@ -169,6 +169,31 @@ spec:
             hidden:
               description: Makes the data stream hidden
               type: boolean
+    route:
```

Comment:
As I wanted to remove this new key `routing_rules` in case spec version is < 2.9.0, I had to do in this way (adding it to the manifest).

Adding it as a new file, I would have to remove the entry also in the `spec.yml` but I think it is not supported json patches in those files.

Reply:
Ah yes, but well, we should give priority to how we want packages to look like. If this is the only reason to keep this in the manifest, lets add support for json patches in the folder spec files.

Reply:
I'll check how to take into account JSON patches in folder spec files

Reply:
There is a possible drawback of moving this to its own file.
If `routing_rules` are defined in it own file, the condition to set as required the `dataset` field , I think it cannot be achieved. They would be in different files.

Referring to this block:
```yaml
  allOf:
    - if:
        required:
          - routing_rules
      then:
        required:
          - dataset
```

Reply:
Added a new validation rule in code for this requirement, since it cannot be achieved with if-then-else block in JSON schema.